CMakeLists.txt

```diff
@@ -0,0 +1,161 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(material_handling_robot)
```

Comment:
Change the repository name to match with "material_handling_robot", case sensitive.